src/service/resources-impl.js

```diff
@@ -786,6 +798,35 @@ export class ResourcesImpl {
     }
   }
 
+  /**
+   * Returns the percent of resources that were laid out but never entered viewport.
+   * @return {number}
+   */
+  getEagerElementRatio() {
+    if (this.totalLayoutCount_ === 0) {
+      return 0;
+    }
+
+    let eagerCount = 0;
+    for (const r of this.resources_) {
```

Comment:
`for...of` not supported on IE: https://caniuse.com/#feat=mdn-javascript_statements_for_of

Reply:
i've definitely made this mistake before. with es6 features I usually assume we polyfill something like this that at first seems super simple, but then i remember it requires iterator